chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/reviewer/SurveillanceDetailsReviewer.java

```diff
@@ -0,0 +1,145 @@
+package gov.healthit.chpl.validation.surveillance.reviewer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+
+import gov.healthit.chpl.dao.CertifiedProductDAO;
+import gov.healthit.chpl.dao.surveillance.SurveillanceDAO;
+import gov.healthit.chpl.domain.CertifiedProduct;
+import gov.healthit.chpl.domain.surveillance.Surveillance;
+import gov.healthit.chpl.domain.surveillance.SurveillanceNonconformity;
+import gov.healthit.chpl.domain.surveillance.SurveillanceRequirement;
+import gov.healthit.chpl.domain.surveillance.SurveillanceType;
+import gov.healthit.chpl.dto.CertifiedProductDetailsDTO;
+import gov.healthit.chpl.entity.surveillance.SurveillanceEntity;
+import gov.healthit.chpl.exception.EntityRetrievalException;
+import gov.healthit.chpl.util.ChplProductNumberUtil;
+import gov.healthit.chpl.util.ErrorMessageUtil;
+
+@Component
+public class SurveillanceDetailsReviewer implements Reviewer {
+
+    private ErrorMessageUtil msgUtil;
+    private CertifiedProductDAO cpDao;
+    private SurveillanceDAO survDao;
+    private ChplProductNumberUtil chplProductNumberUtil;
+
+    @Autowired
+    public SurveillanceDetailsReviewer(CertifiedProductDAO cpDao, SurveillanceDAO survDao,
+            ChplProductNumberUtil chplProductNumberUtil, ErrorMessageUtil msgUtil) {
+        this.cpDao = cpDao;
+        this.survDao = survDao;
+        this.chplProductNumberUtil = chplProductNumberUtil;
+        this.msgUtil = msgUtil;
+    }
+
```

Comment:
This is just a preference note, not a required change. I'm not sure what we've decided as a team but: I like to see @Override annotations when a method is overridden such as here here, from "void review(Surveillance surveillance)." It helps identify the difference and makes reading/understanding the code faster.